packages/web-components/fast-components/src/skeleton/fixtures/base.html

```diff
@@ -1,32 +1,23 @@
 <style>
-    .card-placeholder {
-        background-color: #fff;
-        padding: 20px;
-        width: 500px;
-        border-radius: 4px;
+    fast-card {
+        --card-width: 500px;
+        --card-height: auto;
     }
 
-    .card-pattern {
-        border-radius: 4px;
-        width: 500px;
-        height: 250px;
+    .card-padding {
+        padding: 20px;
     }
-
     @media (forced-colors: active) {
-        .card-placeholder {
-            background-color: ButtonText;
-        }
-
-        .card-pattern {
-            background-color: #000;
+        fast-card {
+            background-color: buttontext;
         }
     }
 </style>
 
 <h1>Skeleton</h1>
 
-<h2>Used as element blocks</h2>
-<div class="card-placeholder">
+<h4>Used as element blocks</h4>
```

Comment:
What did we opt for heading level 4 here and below?

Reply:
that should stay h2